docs/guide/overview-of-event-sources.md

```diff
@@ -140,3 +147,47 @@ functions:
           topic_name: aggregate
           display_name: Data aggregation pipeline
 ```
+
+### Kinesis Streams
+
+Currently there's no native support for Kinesis Streams ([we need your feedback](https://github.com/serverless/serverless/issues/1608))
+but you can use custom provider resources to setup the mapping:
+
+```yml
+# serverless.yml
+
+resources
+  Resources:
+    mapping:
+      Type: AWS::Lambda::EventSourceMapping
+      Properties:
+        BatchSize: 10
+        EventSourceArn: "arn:aws:kinesis:<region>:<aws-account-id>:stream/<stream-name>"
+        FunctionName:
+          Fn::GetAtt:
+            - "<function-name>"
+            - "Arn"
+        StartingPosition: "TRIM_HORIZON"
+```
+
+### DynamoDB Streams
+
+Currently there's no native support for DynamoDB Streams ([we need your feedback](https://github.com/serverless/serverless/issues/1441))
+but you can use custom provider resources to setup the mapping:
+
+```yml
+# serverless.yml
+
+resources
+  Resources:
+    mapping:
+      Type: AWS::Lambda::EventSourceMapping
+      Properties:
+        BatchSize: 10
+        EventSourceArn: "arn:aws:dynamodb:<region>:<aws-account-id>:table/<table-name>/stream/<stream-name>"
```

Comment:
same comment as above, although I'm not sure you can create a dynamoDB stream as a CF resource. Hmmm. I'd just mention it for kinesis stream and leave this as it is.